docs/direct/api/v1.md

```diff
@@ -1485,490 +1862,2635 @@ The properties of each endpoint type are listed below.
 
-Returns a list of all phone numbers on a customer account.
+#### Total Duration
+`GET customers/{customerId}/calls/stats?`
+
+
+<Tabs>
+  <TabItem value="json" label="Example Request">
 
-Example request:
 
 ```
-GET https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers?pageSize=2&page=1
-```
+GET https://my.nimvelo.com/api/v1/customers/5/calls/stats?startedAfter=2023-05-21T21:00:00.000Z&startedBefore=2023-05-28T20:59:59.999Z&timeZone=%2B03%3A00&metric=duration
+
+``` 
+
 
-```response
+
+
+
+  </TabItem>
+  <TabItem  value="result" label="Example Response">
+
+```json
 HTTP/1.1 200 OK
 
 {
-  "totalItems" : 15,
-  "pageSize" : 2,
-  "page" : 1,
-  "items" : [ {
-    "type" : "did",
-    "uri" : "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/47",
-    "created" : "2014-03-10T13:37:00.000+0000",
-    "number" : "01212854400",
-    "identifier" : "Main",
-    "smsEnabled" : true,
-    "smsAllowIncoming" : true,
-    "smsNotificationEmail" : "contact@fredbloggs.com",
-    "smsNotificationUrl" : "https://example.com/sms",
-    "faxEnabled" : false
-  }, {
-    "type" : "did",
-    "uri" : "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers/50",
-    "created" : "2014-03-10T13:37:00.000+0000",
-    "number" : "01212854411",
-    "identifier" : "",
-    "destination" : "https://pbx.sipcentric.com/api/v1/customers/25/endpoints/101",
-    "smsEnabled" : true,
-    "smsAllowIncoming" : true,
-    "smsNotificationEmail" : "support@sipcentric.com",
-    "smsNotificationUrl" : "https://example.com/sms",
-    "faxEnabled" : false,
-    "routingRules" : [ {
-      "timeInterval" : "https://pbx.sipcentric.com/api/v1/customers/25/timeintervals/594",
-      "destination" : "https://pbx.sipcentric.com/api/v1/customers/25/endpoints/102"
-    }, {
-      "timeInterval" : "https://pbx.sipcentric.com/api/v1/customers/25/timeintervals/625",
-      "destination" : "https://pbx.sipcentric.com/api/v1/customers/25/endpoints/102"
-    }, {
-      "timeInterval" : "https://pbx.sipcentric.com/api/v1/customers/25/timeintervals/704",
-      "destination" : "https://pbx.sipcentric.com/api/v1/customers/25/endpoints/102"
-    } ]
-  } ],
-  "nextPage" : "https://pbx.sipcentric.com/api/v1/customers/25/phonenumbers?pageSize=2&page=2"
+  "type": "datatable",
+  "uri": "https://my.nimvelo.com/api/v1/customers/5/calls/stats?startedAfter=2023-05-21T21:00:00.000Z&startedBefore=2023-05-28T20:59:59.999Z&timeZone=%2B03%3A00&metric=duration",
+  "columnHeaders": [
+    {
+      "name": "duration",
+      "columnType": "METRIC"
+    }
+  ],
+  "rows": [
+    [
+      108.0
+    ]
+  ],
+  "totals": {
+    "duration": 108.0
+  }
 }
 ```
+ </TabItem>
+</Tabs>
 
-#### Get phone number
+**Explanation:**
+From the above results we see that the total Duration for account 5 is 108.0 seconds. on the Web ui this will be represented as *1 minute and 48 seconds*
```

Comment:
Formatting, capitalisation.

Reply:
Do we need explanations? Seeing a few of these, maybe we should remove and / or discuss first.

Reply:
No we don't, if we want to have guides on how/why to use endpoints, that can be separate. Also they don't exist elsewhere. Removed.